zrml/prediction-markets/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [dependencies]
-frame-benchmarking = { branch = "moonbeam-polkadot-v0.9.17", default-features = false, optional = true, git = "https://github.com/purestake/substrate" }
-frame-support = { branch = "moonbeam-polkadot-v0.9.17", default-features = false, git = "https://github.com/purestake/substrate" }
-frame-system = { branch = "moonbeam-polkadot-v0.9.17", default-features = false, git = "https://github.com/purestake/substrate" }
-orml-traits = { default-features = false, git = "https://github.com/PureStake/open-runtime-module-library" }
-parity-scale-codec = { default-features = false, features = ["derive", "max-encoded-len"], version = "2.0.0" }
-sp-arithmetic = { branch = "moonbeam-polkadot-v0.9.17", default-features = false, git = "https://github.com/purestake/substrate" }
-sp-runtime = { branch = "moonbeam-polkadot-v0.9.17", default-features = false, git = "https://github.com/purestake/substrate" }
+frame-benchmarking = { branch = "moonbeam-polkadot-v0.9.19", default-features = false, optional = true, git = "https://github.com/purestake/substrate" }
+frame-support = { branch = "moonbeam-polkadot-v0.9.19", default-features = false, git = "https://github.com/purestake/substrate" }
+frame-system = { branch = "moonbeam-polkadot-v0.9.19", default-features = false, git = "https://github.com/purestake/substrate" }
+orml-traits = { branch = "polkadot-0.9.19-latest", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
```

Comment:
We are accessing a zeitgeistpm clone directly instead of using a patch. Is this intended?

Reply:
The reason was because we fork off [open-web3-stack/open-runtime-module-library](https://github.com/open-web3-stack/open-runtime-module-library), but we need [PureStake/open-runtime-module-library/](https://github.com/PureStake/open-runtime-module-library/) and I have no permissions to delete the existing fork (which blocks forking off from PureStake). 
We need the same branch `moonbeam-polkadot-v0.9.19` and commit history as in [PureStake/open-runtime-module-library/](https://github.com/PureStake/open-runtime-module-library/)  within our fork, such that the patch works. But since we fork off [open-web3-stack/open-runtime-module-library](https://github.com/open-web3-stack/open-runtime-module-library), the branch and commit history don't exist. Instead of replicating them on our fork and merging the newer changes from orml in, I decided to just directly branch of the desired commit in our orml fork and redirect the dependencies to `PureStake`'s forks.
Maybe there is a better approach than this, but I am not aware of it. Anyways, if we forget to update those dependencies during the next codebase upgrade, the code won't compile. So pointing them again to the ORML fork of PureStake during the next codebase upgrade is inevitable.